packages/react-components/theme-designer/src/components/Swatch/Swatch.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import { makeStyles, mergeClasses } from '@griffel/react';
+import {
+  tokens,
+  TabList,
+  Tab,
+  Input,
+  Button,
+  Dropdown,
+  Option,
+  Slider,
+  Badge,
+  Switch,
+  Radio,
+  RadioGroup,
+  Checkbox,
+  Field,
+  Persona,
+  useId,
+  Link,
+} from '@fluentui/react-components';
+import {
+  SearchRegular,
+  bundleIcon,
+  MeetNowRegular,
+  MeetNowFilled,
+  CalendarLtrFilled,
+  CalendarLtrRegular,
+} from '@fluentui/react-icons';
+
+export interface ContentProps {
+  className?: string;
```

Comment:
Hmmm is this a legacy prop? I don't know that it's being used so we can probably remove this all together.